pkg/agent/controller/agent.go

```diff
@@ -154,6 +154,24 @@ func New(spec *AgentSpecification, syncerConf broker.SyncerConfig, kubeClientSet
 		return nil, err
 	}
 
+	if agentController.globalnetEnabled {
+		agentController.gipSyncer, err = syncer.NewResourceSyncer(&syncer.ResourceSyncerConfig{
+			Name:             "GlobalIngressIP -> ServiceImport",
+			SourceClient:     syncerConf.LocalClient,
+			SourceNamespace:  metav1.NamespaceAll,
+			RestMapper:       syncerConf.RestMapper,
+			Federator:        agentController.serviceImportSyncer.GetLocalFederator(),
+			ResourceType:     GetGlobalIngressIPObj(),
+			Transform:        agentController.globalIngressIPToServiceImport,
+			OnSuccessfulSync: agentController.onSuccessfulServiceImportSync,
+			Scheme:           syncerConf.Scheme,
+		})
+
+		if err != nil {
+			klog.Fatalf("Error creating globalIngressIP Syncer: %s", err.Error())
```

Comment:
To consistent with the other error paths.

```suggestion
			return nil, err
```